src/nginxconfig/i18n/en/common.js

```diff
@@ -35,6 +35,7 @@ export default {
 	https: 'HTTPS',
 	letsEncrypt: 'Let\'s Encrypt',
 	python: 'Python',
+	docker: 'Dockerfile',
```

Comment:
```suggestion
	docker: 'Docker',
```
Lets keep the tab named 'Docker' so it can potentially be used for more in the future